README.md

```diff
@@ -196,6 +196,7 @@ python aria_streamer.py --data-path="/path/to/directory/containing/vrs/and/mps/f
 3. `--dry-run` (Input: bool): This is used to indicate if it should command robot to move or not. If dry run is true, then it will not send the commands to the robot to make it move. *But this will still require you to have robot watching the QR code for localizing itself in the system*.
 4. `--verbose` (Input: bool): This flag is for updating the visual stream with detections from both QR & object detectors.
 
+
```

Comment:
@tusharsangam this line change seems unncessary. Its causing an extra file change. Let's revert this.